WordPress/Classes/Utility/Logging/WPCrashLoggingProvider.swift

```diff
@@ -48,12 +54,20 @@ struct WPCrashLoggingDataProvider: CrashLoggingDataProvider {
     }
 
     var currentUser: TracksUser? {
-        let context = ContextManager.sharedInstance().mainContext
-
-        guard let account = try? WPAccount.lookupDefaultWordPressComAccount(in: context) else {
-            return nil
+        return contextManager.performQuery { context -> TracksUser? in
+            guard let account = try? WPAccount.lookupDefaultWordPressComAccount(in: context) else {
+                return nil
+            }
+            return TracksUser(userID: account.userID.stringValue, email: account.email, username: account.username)
         }
+    }
+}
+
+extension NSManagedObjectContext {
 
-        return TracksUser(userID: account.userID.stringValue, email: account.email, username: account.username)
+    func proxy_save() throws {
```

Comment:
I believe this extension was added unintentionally?

Reply:
@crazytonyli Thats correct, thanks for the catch. 🙇

Reply:
Resolved in https://github.com/wordpress-mobile/WordPress-iOS/pull/20846/commits/119b6d148cf3a2b9b7ed5cb4b8174bac4ed7ffab